web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -127,7 +127,7 @@ const ApiHelpers = (pathPrefix, defaultMetricsWindow = '1m') => {
     // Traffic Performance Summary. This retrieves stats for the given resource.
     let resourceUrl = '/api/tps-reports?resource_type=' + type;
 
-    if (_isEmpty(namespace)) {
+    if (_isEmpty(namespace) || namespace === "all") {
```

Comment:
For consistency, I think `urlForResourceNoStats` below also requires the addition of `|| namespace === "all"`